Sources/Fuzzilli/FuzzIL/AbstractInterpreter.swift

```diff
@@ -351,6 +351,9 @@ public struct AbstractInterpreter {
         case let op as CallMethod:
             set(instr.output, inferMethodSignature(of: op.methodName, on: instr.input(0)).outputType)
 
+        case is CallComputedMethod:
+            set(instr.output, inferMethodSignature(of: instr.input(1).description, on: instr.input(0)).outputType)
```

Comment:
This doesn't quite work. Without an additional analyzer that can determine the runtime value of a variable this won't be possible, so I think you can just set the result to `.unknown` directly, similar to how e.g. [`LoadComputedProperty` is handled](https://github.com/googleprojectzero/fuzzilli/blob/40a119bdba15f764b96330c19a3d9de55516459d/Sources/Fuzzilli/FuzzIL/AbstractInterpreter.swift#L344). Feel free to make a TODO comment about improving this eventually if you want though